src/api/api.js

```diff
@@ -0,0 +1,23 @@
+const URL = 'https://us-central1-bookstore-api-e63c8.cloudfunctions.net/bookstoreApi/apps/KuV7Z2YFTRBTq309Vw2X/books';
+
+export const getAllBooks = () => fetch(URL).then((response) => {
+  if (response.ok) {
+    return response.json();
+  }
+  return {};
+});
```

Comment:
Users after adding books and refreshing the page can see only the first book added in the array. Please make sure all the books are shown on the page.